### ISSUE TITLE
"Acquire team models for completions" setting not found

### ISSUE THREAD

**Issue by bddckr** [MicrosoftDocs/intellicode#141]
I cannot find the mentioned setting in my installation of Visual Studio. It's not listed in `Tools > Options > IntelliCode`:
![image](https://user-images.githubusercontent.com/294347/69006169-a4b83200-092b-11ea-800d-ee03659d0558.png)

<details><summary>Visual Studio 16.3.9</summary>

```
Microsoft Visual Studio Community 2019
Version 16.3.9
VisualStudio.16.Release/16.3.9+29509.3
Microsoft .NET Framework
Version 4.8.03752

Installed Version: Community

Visual C++ 2019   00435-60000-00000-AA534
Microsoft Visual C++ 2019

.ignore   1.2.77
A language service that makes it painless to handle all types of .ignore files such as .gitignore, .tfignore, etc.

Add New File   3.5.134
The fastest and easiest way to add new files to any project - including files that start with a dot

ASP.NET and Web Tools 2019   16.3.286.43615
ASP.NET and Web Tools 2019

Azure App Service Tools v3.0.0   16.3.286.43615
Azure App Service Tools v3.0.0

C# Tools   3.3.1-beta3-19461-02+2fd12c210e22f7d6245805c60340f6a34af6875b
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

CheckedExceptionsPackage Extension   1.0
CheckedExceptionsPackage Visual Studio Extension Detailed Info

CodeMaid   11.1.209
CodeMaid is an open source Visual Studio extension to cleanup and simplify our C#, C++, F#, VB, PHP, PowerShell, R, JSON, XAML, XML, ASP, HTML, CSS, LESS, SCSS, JavaScript and TypeScript coding.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

EditorConfig Language Service   1.17.260
Language service for .editorconfig files.

EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs.

Extensibility Message Bus   1.2.0 (d16-2@8b56e20)
Provides common messaging-based MEF services for loosely coupled Visual Studio extension components communication and integration.

File Icons   2.7
Adds icons for files that are not recognized by Solution Explorer

FormatCommand Extension   1.0
FormatCommand Visual Studio Extension Detailed Info

IntelliCode Extension   1.0
IntelliCode Visual Studio Extension Detailed Info

JetBrains ReSharper Ultimate 2019.2.3   Build 192.0.20191016.60459
JetBrains ReSharper Ultimate package for Microsoft Visual Studio. For more information about ReSharper Ultimate, visit http://www.jetbrains.com/resharper. Copyright © 2019 JetBrains, Inc.

Line Press   13d97f3e-75e2-4715-a444-715b5f39f0a4
Control the line-height of code lines, and compress empty or any unimportant lines. Have best of both worlds, more code view with no cram.

Microsoft Continuous Delivery Tools for Visual Studio   0.4
Simplifying the configuration of Azure DevOps pipelines from within the Visual Studio IDE.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft Library Manager   2.0.83+gbc8a4b23ec
Install client-side libraries easily to any web project

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual C++ Wizards   1.0
Microsoft Visual C++ Wizards

Microsoft Visual Studio Tools for Containers   1.1
Develop, run, validate your ASP.NET Core applications in the target environment. F5 your application directly into a container with debugging, or CTRL + F5 to edit & refresh your app without having to rebuild the container.

Microsoft Visual Studio VC Package   1.0
Microsoft Visual Studio VC Package

Mono Debugging for Visual Studio   16.3.7 (9d260c5)
Support for debugging Mono processes with Visual Studio.

MoveTypeToFile   1.0
MoveTypeToFile

NamespaceFixer   2.6
VS Extension to fix namespaces following C# and VB.NET convention

NuGet Package Manager   5.3.1
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

Open in Visual Studio Code   1.4.46
Adds a menu command that lets you open any solution, project, folder and file in Visual Studio Code.

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

Smart Command Line Arguments Extension   2.2.0
A Visual Studio Extension which aims to provide a better UI to manage your command line arguments. For additional information, visit https://github.com/MBulli/SmartCommandlineArgs

SQL Server Data Tools   16.0.61908.27190
Microsoft SQL Server Data Tools

SteroidsVS   1.0
Multi-purpose extension for VisualStudio

SubwordNavigation   1.5.0
Provides subword navigation for Visual Studio.

TypeScript Tools   16.0.10821.2002
TypeScript Tools for Microsoft Visual Studio

Viasfora   4.2.188
Add color to your Visual Studio editor!

Visual Basic Tools   3.3.1-beta3-19461-02+2fd12c210e22f7d6245805c60340f6a34af6875b
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools 10.4 for F# 4.6   16.3.0-beta.19455.1+0422ff293bb2cc722fe5021b85ef50378a9af823
Microsoft Visual F# Tools 10.4 for F# 4.6

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

Visual Studio Spell Check Everywhere   VSSpellCheckEverywhere
An extension that enables spell checking within any Visual Studio file editor or tool window that uses WPF text boxes.
https://GitHub.com/EWSoftware/VSSpellChecker

Visual Studio Spell Checker   VSSpellChecker
An editor extension that checks the spelling of comments, strings, and plain text as you type or interactively with tool windows.
https://GitHub.com/EWSoftware/VSSpellChecker

Visual Studio Tools for CMake   1.0
Visual Studio Tools for CMake

Visual Studio Tools for CMake   1.0
Visual Studio Tools for CMake

Visual Studio Tools for Containers   1.0
Visual Studio Tools for Containers

Visual Studio Tools for Unity   4.3.3.0
Visual Studio Tools for Unity

VisualStudio.Mac   1.0
Mac Extension for Visual Studio

VSColorOutput   2.6.7
Color output for build and debug windows - https://mike-ward.net/vscoloroutput

Xamarin   16.3.0.278 (d16-3@40034cd)
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin Designer   16.3.0.256 (remotes/origin/d16-3@8a223bfd7)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   16.3.565 (27e9746)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   10.0.6.2 (d16-3/c407838)
Xamarin.Android Reference Assemblies and MSBuild support.
    Mono: mono/mono/2019-06@476d72b9e32
    Java.Interop: xamarin/java.interop/d16-3@5836f58
    LibZipSharp: grendello/LibZipSharp/d16-3@71f4a94
    LibZip: nih-at/libzip/rel-1-5-1@b95cf3fd
    ProGuard: xamarin/proguard/master@905836d
    SQLite: xamarin/sqlite/3.27.1@8212a2d
    Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-3@cb41333


Xamarin.iOS and Xamarin.Mac SDK   13.6.0.12 (e3c2b40)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.
```
</details>

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c2089e9f-f405-f6a4-2bf2-9da8acfe2885
* Version Independent ID: 56ce5868-6ade-0d11-0038-068ffbcd6841
* Content: [IntelliSense based on your code - Visual Studio IntelliCode](https://docs.microsoft.com/en-us/visualstudio/intellicode/custom-models)
* Content Source: [docs/custom-models.md](https://github.com/MicrosoftDocs/intellicode/blob/master/docs/custom-models.md)
* Product: **visual-studio-family**
* Technology: **intellicode**
* GitHub Login: @markw-t
* Microsoft Alias: **mwthomas**

**Comment by pgroene** [comment]
Thank you for taking the time to report this issue. This preview feature is available from  Visual Studio 16.4 Preview 2. You can get it at the preview channel: https://visualstudio.microsoft.com/vs/preview/

We will update the documentation to make it clear. Sorry for the unconvincing.

--
Peter [MSFT]

**Comment by bddckr** [comment]
No problem, thanks for clarifying!

**Comment by markw-t** [comment]
The feature has now released (in Visual Studio 16.8) and is no longer in preview.
